client/landing/gutenboarding/onboarding-block/vertical-select/index.tsx

```diff
@@ -101,24 +101,44 @@ export default function VerticalSelect( { inputClass }: Props ) {
 
 	return (
 		<div className="vertical-select">
-			<input
-				ref={ inputRef }
-				className={ inputClass }
-				placeholder={ NO__( 'enter a topic' ) }
-				onChange={ handleSuggestionChangeEvent }
-				onFocus={ showSuggestions }
-				onBlur={ hideSuggestions }
-				onKeyDown={ handleSuggestionKeyDown }
-				autoComplete="off"
-				value={ value }
-			/>
-			{ suggestionsVisibility && (
-				<Suggestions
-					ref={ suggestionRef }
-					query={ inputValue }
-					suggestions={ ! verticals.length ? loadingMessage : suggestions }
-					suggest={ handleSelect }
-				/>
+			{ suggestionsVisibility || ! value ? (
+				<>
+					<input
+						ref={ inputRef }
+						className={ inputClass }
+						placeholder={ NO__( 'enter a topic' ) }
+						onChange={ handleSuggestionChangeEvent }
+						onFocus={ showSuggestions }
+						onBlur={ hideSuggestions }
+						onKeyDown={ handleSuggestionKeyDown }
+						onMouseDown={ onClick }
+						autoComplete="off"
+						value={ value }
+					/>
+					{ ! forceHideSuggestions && (
+						<Suggestions
+							ref={ suggestionRef }
+							query={ inputValue }
+							suggestions={ ! verticals.length ? loadingMessage : suggestions }
+							suggest={ handleSelect }
+						/>
+					) }
+				</>
+			) : (
+				<span
```

Comment:
This is a little bit weird, but without this, the text for some of the categories doesn't fit within the input width. This way, when the user is not interacting with the component, the current value gets rendered within a span that deals better with long values.